new marketo testing account added [BACK-1529]

ginnyyadav opened the pull request:
The orginal marketo account for testing the connection to shoreline was accidentlly removed from marketo. Created a new one.

ginnyyadav committed:
new marketo testing account added

jh-bate reviewed (commented):
just to add my orginal comment to the PR 

> maybe it should be a param passed in similar to user names and PW, that should come from config would be my argument

ginnyyadav committed:
remove marketo tests

these marketo tests are too fragile and  are being replaced by https://grafana.production.tidepool.org/d/5sv7jfiGk/shoreline?orgId=1 and https://tidepool.atlassian.net/browse/BACK-1534?atlOrigin=eyJpIjoiNTQzYTVkYmRjODRjNDQwMzlhNmE5MTc3MjBlNWM3MDkiLCJwIjoiamlyYS1zbGFjay1pbnQifQ

ginnyyadav commented:
these marketo tests are too fragile and  are being replaced by https://grafana.production.tidepool.org/d/5sv7jfiGk/shoreline?orgId=1 and https://tidepool.atlassian.net/browse/BACK-1534?atlOrigin=eyJpIjoiNTQzYTVkYmRjODRjNDQwMzlhNmE5MTc3MjBlNWM3MDkiLCJwIjoiamlyYS1zbGFjay1pbnQifQ

jh-bate reviewed (approved):
LGTM - yeah fragile tests don't buy us much :)
